src/Native/Bootstrap/main.cpp

```diff
@@ -27,11 +25,7 @@ NoStl::vector<void*> __myVector;
 // On Windows, the #pragma ... /merge directive folds the book-end sections and all .modules$I sections from all input
 // obj files into .rdata in alphabetical order.
 //
-// On Linux, a linker script is used (see src/scripts/linkerscript) to do the equivalent since there is no similar
```

Comment:
Yes, it does. But the linker script is currently referred to by the dotnet build that builds the tests, so we cannot remove it until we replace it with a new build tool.